README.md

```diff
@@ -4,10 +4,41 @@
 
 This application demonstrates a multi-party video application built with [twilio-video.js](https://github.com/twilio/twilio-video.js) and [Create React App](https://github.com/facebook/create-react-app).
 
+![App Preview](https://user-images.githubusercontent.com/12685223/76361972-c035b700-62e5-11ea-8f9d-0bb24bd73fd4.png)
+
+## Features
+
+- Video conferencing with real-time video and audio
+- Enable/disable camera
+- Mute/unmute microphone
+- Screen sharing
+- [Dominant speaker](https://www.twilio.com/docs/video/detecting-dominant-speaker) detection
+- [Network quality](https://www.twilio.com/docs/video/using-network-quality-api) indicator
```

Comment:
this app also supports bandwidth profile api

Reply:
Added